spanner/session.go

```diff
@@ -1178,14 +1355,32 @@ func (p *sessionPool) incNumInUse(ctx context.Context) {
 
 func (p *sessionPool) incNumInUseLocked(ctx context.Context) {
 	p.numInUse++
-	p.recordStat(ctx, SessionsCount, int64(p.numInUse), tagNumInUseSessions)
-	p.recordStat(ctx, AcquiredSessionsCount, 1)
+	p.recordStat(ctx, SessionsCount, int64(p.numInUse), tagNumInUseSessions, tag.Tag{Key: tagKeyIsMultiplexed, Value: "false"})
+	p.recordStat(ctx, AcquiredSessionsCount, 1, tag.Tag{Key: tagKeyIsMultiplexed, Value: "false"})
 	if p.otConfig != nil {
-		p.recordOTStat(ctx, p.otConfig.acquiredSessionsCount, 1)
+		p.recordOTStat(ctx, p.otConfig.acquiredSessionsCount, 1, attributeKeyIsMultiplexed.String("false"))
 	}
 	if p.numInUse > p.maxNumInUse {
 		p.maxNumInUse = p.numInUse
-		p.recordStat(ctx, MaxInUseSessionsCount, int64(p.maxNumInUse))
+		p.recordStat(ctx, MaxInUseSessionsCount, int64(p.maxNumInUse), tag.Tag{Key: tagKeyIsMultiplexed, Value: "false"})
+	}
+}
+
+func (p *sessionPool) incNumMultiplexedInUse(ctx context.Context) {
+	p.mu.Lock()
+	p.incNumMultiplexedInUseLocked(ctx)
+	p.mu.Unlock()
+}
+func (p *sessionPool) incNumMultiplexedInUseLocked(ctx context.Context) {
```

Comment:
Can we combine this method with the one for regular sessions? They seem to have largely the same implementation.

Reply:
I want to keep it separate for readability from parts of code just that one is aware what tags are passed from different type of ops